olclient/openlibrary.py

```diff
@@ -1008,6 +1012,8 @@ def first(self):
     class Document(object):
         """An aggregate OpenLibrary Work summarizing all Editions of a Book"""
 
+        OL = OpenLibrary()
```

Comment:
This shouldn't be needed, and is what is causing the tests to fail since Travis doesn't have any keys set up.

Reply:
Yeah, this was the inclusion that was causing the TravisCI checks to fail.
I have handled it in commit:[8536743 ](https://github.com/internetarchive/openlibrary-client/pull/133/commits/853674338150a57b96cce29fb0e661b781926406)